src/main/java/db/migration/V192__Assign_Instructor_Types.java

```diff
@@ -0,0 +1,225 @@
+package db.migration;
+
+import org.flywaydb.core.api.migration.jdbc.JdbcMigration;
+
+import java.sql.*;
+
+public class V192__Assign_Instructor_Types implements JdbcMigration {
+    static Long INSTRUCTOR_ROLE = 15L;
+
+    static Long EMERITI = 1L;
+    static Long VISITING_PROFESSOR = 2L;
+    static Long ASSOCIATE_INSTRUCTOR = 3L;
+    static Long UNIT_18_PRE_SIX = 4L;
+    static Long CONTINUING_LECTURER = 5L;
+    static Long LADDER_FACULTY = 6L;
+    static Long INSTRUCTOR = 7L;
+
+    /**
+     * For every instructorTypeCost, looks at the deprecrated description field and associate to the matching instructorType.
+     * Will also delete instructorTypeCosts with no amount set.
+     * This is in preparation for dropping the description column and making instructorTypeCosts optional.
+     *
+     * @param connection
+     * @throws Exception
+     */
+    @Override
+    public void migrate(Connection connection) throws Exception {
+        PreparedStatement psInstructorTypeCosts = connection.prepareStatement("SELECT * FROM `InstructorTypeCosts`");
+
+        try {
+            connection.setAutoCommit(false);
+
+            ResultSet rsInstructorTypeCosts = psInstructorTypeCosts.executeQuery();
+
+            while (rsInstructorTypeCosts.next()) {
+                Long id = rsInstructorTypeCosts.getLong("Id");
```

Comment:
Given that there are multiple 'ids' in this file, instructotTypeCostId might be a better choice for a name.